### DISCUSSION TITLE
show gif

### DISCUSSION THREAD

**Discussion by Pierrot66657** [mathoudebine/turing-smart-screen-python#22] [Ideas]
Hello, I am looking to display a Gif, if anyone has the solution?

Thanks

**Comment by mathoudebine** [comment]
Hello @Pierrot66657 !
The screen refresh rate is very slow (takes approx. 4 seconds to load an image full screen) so displaying a gif would not be very responsive
Even if you display a gif to a small part of the screen, it would still take some time between each frame (~1s if the gif takes 1/3 of the screen) so i'm not sure this display is made for this usage.

**Comment by Pierrot66657** [comment]
ok thanks for the answer, i understand
